learn/experimental/autobatching.md

```diff
@@ -0,0 +1,42 @@
+# Auto-batching
+
+If enabled, multiple document addition requests will be automatically combined into a batch and processed together to speed up indexation.
+
+We would like your feedback on this feature! [Join the discussion](https://github.com/meilisearch/meilisearch/discussions/2070).
+
+## Enable auto-batching
+
+To enable auto-batching, start Meilisearch while supplying the `--enable-autobatching` CLI flag:
+
+```
+./meilisearch --enable-autobatching
```

Comment:
Once again registering my mild annoyance at coexisting hyphen/no hyphen spellings of the feature.

Reply:
Hi @guimachiavelli! I will check if we can name it `--enable-auto-batching`. If I remember @dichotommy also pointed it.